coral/src/app/features/topics/acl-request/forms/TopicConsumerForm.tsx

```diff
@@ -103,7 +103,7 @@ const TopicConsumerForm = ({
     <>
       {isError && (
         <Box marginBottom={"l1"} role="alert">
-          <Alert type="warning">{parseErrorMsg(error)}</Alert>
+          <Alert type="error">{parseErrorMsg(error)}</Alert>
```

Comment:
Not wrapping this one is a `role="alert"` div? :o

Actually it seems we are inconsistently doing this 🤔 Should we have a thin wrapper over `Alert` so that we always have it?

Reply:
It is an alert :D 
`<Box marginBottom={"l1"} role="alert">`

(easy to overlook, happened to me too)

>Should we have a thin wrapper over Alert so that we always have it?
I would rather throw a PR at the DS so it can be a added directly in `<Alert>` -> dependent on the context where it happens, we may not want it to be an actual `alert` but maybe a kind of non-assertive dialog.